cocos/ui/UIEditBox/Mac/CCUIEditBoxMac.mm

```diff
@@ -135,6 +135,8 @@ - (NSWindow *)window
 - (void)openKeyboard
 {
     [self.window.contentView addSubview:self.textInput];
+    //FIXME: it will cause a few OpenGL error once, should be fix when glfw version upgraded.
```

Comment:
@minggo  
Already opened a ticket in  https://github.com/glfw/glfw/issues/892  But it seems not get active response.